tools/cldr-code/src/main/java/org/unicode/cldr/util/Emoji.java

```diff
@@ -74,34 +75,28 @@ public class Emoji {
 
     static {
         /*
-           # group: Smileys & People
-           # subgroup: face-positive
-           1F600 ; fully-qualified     # 😀 grinning face
-        */
+         * Example from emoji-test.txt:
+         *   # group: Smileys & Emotion
+         *   # subgroup: face-smiling
+         *   1F600 ; fully-qualified # 😀 grinning face
+         */
         Splitter semi = Splitter.on(CharMatcher.anyOf(";#")).trimResults();
         String majorCategory = null;
         String minorCategory = null;
-        long majorOrder = 0;
-        long minorOrder = 0;
```

Comment:
majorOrder and minorOrder were assigned to but unused, resulting in compiler warnings; cleaned up